urllib3 is part of port py-requests, not a separate module installation
--- requirements.txt.orig	2016-04-13 01:54:44.000000000 +0200
+++ requirements.txt	2016-04-13 01:54:52.000000000 +0200
@@ -1,2 +1 @@
 requests>=1.1
-urllib3>=1.8
